api/src/main/java/com/datatorrent/api/Context.java

```diff
@@ -393,6 +393,10 @@
      */
     Attribute<String> CONTAINER_JVM_OPTIONS = new Attribute<>(String2String.getInstance());
     /**
+     * The options of dynamic apex logger appender
+     */
+    Attribute<String> LOGGER_APPENDER = new Attribute<>(String2String.getInstance());
```

Comment:
@PramodSSImmaneni I think that SPIN_MILLIS is not a good example as PortContext.SPIN_MILLIS is not used and should be deprecated. I'd prefer to avoid sharing the same name in multiple Context to avoid confusion.